### ISSUE TITLE
No earnings during sessions 

### ISSUE THREAD

**Issue by isaackielma** [mysteriumnetwork/node#4484]
**Describe the bug**
I started my node allmost 2 weeks now.
All seems ok, I have green lights and doing traffic,
but I have no earnings.

My withdraw is under 0.01 MYST and this from day 1.
Am I missing something or doing something wrong?

**Additional context**

myst@raspberrypi:/var/lib/mysterium-node $ myst account info
[SECTION] General account information:
[INFO] Using identity: 0x4389edc75fc51e2848ecf538e4d4674fa401f67b
[INFO] Registration Status: Registered
[INFO] Channel address: 0x7fc1259bC8fF6B709c7c86238C5999a47EB42E73
[INFO] Balance: 0.000000
» service list
[Running] ID: 89e7ee12-4bb5-4c7b-8578-ba03bca20322 ProviderID: 0x4389edc75fc51e2848ecf538e4d4674fa401f67b Type: wireguard
» service sessions
[Current sessions] 50
[ID: 6291159c-8e93-4f60-807f-a198289fc859] ConsumerID: 0x56da8fb62b3759bfa9d89a09a7aa9846e1922a19 Data: 49.4KiB/20.6KiB Tokens: 0.000000
[ID: 2f8101ab-c8d6-4b4e-8367-b61d42924532] ConsumerID: 0xac83c06eff343ebd3aa137cefca957aa2cf23237 Data: 533.7MiB/20.0MiB Tokens: 0.000000
[ID: c4250909-4e60-4642-ad77-0b6ab69b5dc5] ConsumerID: 0x318f500d5a9e78459684055b911f07e76aa70669 Data: 435.5KiB/415.2KiB Tokens: 0.000000
[ID: ba8ff66c-8515-478f-bb01-230d13b73ddf] ConsumerID: 0xb12009cab006902e57dd762f3b6e3fcbbce17e21 Data: 409.0MiB/14.6MiB Tokens: 0.000000
[ID: c6e95664-c41e-4715-864b-591ad6f20de1] ConsumerID: 0xac83c06eff343ebd3aa137cefca957aa2cf23237 Data: 533.8MiB/20.3MiB Tokens: 0.000000
[ID: cf6cea3f-ebd0-4344-b750-2e827a4e44f2] ConsumerID: 0xc72a9750531f70e0a4ae627073616248b456361d Data: 40.3KiB/15.7KiB Tokens: 0.000000
[ID: 9d6e0744-d96f-4f46-adbe-90c4e664b86c] ConsumerID: 0x18dc1b64c27d4b2f9e23d18418c993da0a3a8922 Data: 427.2KiB/409.9KiB Tokens: 0.000000
[ID: 576256bd-bf7d-4bc5-a71b-9b8ae0b9e938] ConsumerID: 0x6cf6685529664cbb2c800ea7acea687be85f2750 Data: 445.0KiB/437.5KiB Tokens: 0.000000
[ID: c45f2d52-7541-4314-ae20-dd9928cbaac1] ConsumerID: 0x3c2a205f4befaf5653b3abb6fa8c2f165fdba549 Data: 40.0KiB/16.6KiB Tokens: 0.000000
[ID: 9eb2204f-58a7-4d22-9661-824446f1f22b] ConsumerID: 0x03708ef218a5e3f5b1af4479d8d59f8b44f37ccb Data: 40.9KiB/16.2KiB Tokens: 0.000000
[ID: 8cc11388-6a19-4b36-b60c-6f119e52c890] ConsumerID: 0xc863d072dd64ddae92fbc9999cb6c19e6466d9bb Data: 394.0MiB/15.2MiB Tokens: 0.000000
[ID: 4c70046e-a06f-404e-9ae3-0c99d901a143] ConsumerID: 0xebd240db9bcc4d79d2e3113ad5303e622897f1bf Data: 40.5KiB/16.3KiB Tokens: 0.000000
[ID: e1c9f8ad-1c6e-4aa1-9d23-8cdf221ac492] ConsumerID: 0xac96e76e3c9b66e02ba3a87799bde2f6012edff3 Data: 455.7KiB/437.1KiB Tokens: 0.000000
[ID: 65f1cf19-a96b-4931-9058-62b4a29dc72c] ConsumerID: 0xb385326e7b3fdf0f1dc776d67490dea9e583ba18 Data: 2.5MiB/1.4MiB Tokens: 0.000000
[ID: 180c2388-7f4a-4d9e-bc6d-cdef1e20914a] ConsumerID: 0x01ad8799b5fc7844badcdc2143cf492e4b2e164d Data: 295.1MiB/11.1MiB Tokens: 0.000000
[ID: 23ae609c-ee66-41b0-9850-c8a7d69178f4] ConsumerID: 0x3087cd2013bb68550dc488a83cebeb26b5b0ec2b Data: 441.0KiB/416.3KiB Tokens: 0.000000
[ID: 704ce544-1a50-442f-afbf-d96a8b9dc0de] ConsumerID: 0xc9f60877d3609363896c93ac473d5cc0b41ed696 Data: 436.3KiB/417.4KiB Tokens: 0.000000
[ID: 2627804d-01da-4420-b127-71d5fadd5342] ConsumerID: 0x7bac8f70fd69600eb19d6f702c371eeddc336c6c Data: 436.7KiB/400.2KiB Tokens: 0.000000
[ID: beed56ca-5739-447d-ae59-e03be41ae918] ConsumerID: 0xa4bd2c53e2b613cc22591a22e022e311bad28032 Data: 435.1KiB/414.7KiB Tokens: 0.000000
[ID: 3b65c77a-465d-4197-bb9f-c03ccc3b8794] ConsumerID: 0xe88df80ba994a662fec73a104777f018a09d4491 Data: 439.1KiB/413.6KiB Tokens: 0.000000
[ID: aaab2c34-f391-4c83-8c54-9bfe33f18008] ConsumerID: 0x3930c5cfc07fc6ac4b8876dab0751736be6c6333 Data: 1.2MiB/280.6KiB Tokens: 0.000000
[ID: 53c6e592-ec21-45a9-9be0-25a8cf7c8910] ConsumerID: 0x690c4f186d2ad76e6faa0b6f5bf691abf000ed05 Data: 441.7KiB/413.2KiB Tokens: 0.000000
[ID: 742673cc-d3ef-4d63-85b1-a30380ff80ff] ConsumerID: 0xb5a2f38e62c5b3e73717ab0b38379d521f6f5aca Data: 462.1KiB/426.8KiB Tokens: 0.000000
[ID: faee138f-4d68-42c7-afaa-7b36b47c0e14] ConsumerID: 0x652b7ed8143adf41b7be437d490589ccad02bf31 Data: 536.5MiB/20.6MiB Tokens: 0.000000
[ID: c869865e-f1c2-4732-8607-77ca8fefcf4c] ConsumerID: 0xe88df80ba994a662fec73a104777f018a09d4491 Data: 0b/0b Tokens: 0.000000
[ID: 6db3fec6-1d5e-4765-9996-64b98a9250f2] ConsumerID: 0x2d5d016e0d33f7bead532904f16ff99eaad8c716 Data: 424.3MiB/16.2MiB Tokens: 0.000000
[ID: b04cd935-353a-4c6a-84bb-b32a31525843] ConsumerID: 0x4920388f7ca3489fc0ce0a9bb0a2f758e68631f0 Data: 442.0KiB/428.1KiB Tokens: 0.000000
[ID: e800449e-73ee-4248-a6f8-cd1e7b38a37b] ConsumerID: 0x01ad8799b5fc7844badcdc2143cf492e4b2e164d Data: 47.3KiB/18.7KiB Tokens: 0.000000
[ID: b16e4a29-f87f-4499-857d-5275bff4c798] ConsumerID: 0x8f62ffa65258edd6a9702bec95b6927da04fbbbe Data: 532.7MiB/20.6MiB Tokens: 0.000000
[ID: fa132405-21e0-4ead-b2bf-b7ad0fc24ce1] ConsumerID: 0xf8dd8576f86bf1a01f2e00d96defbf1933f7a2b5 Data: 445.9KiB/420.2KiB Tokens: 0.000000
[ID: 0b9827ca-57dc-47f7-80a8-a64bb866c4b9] ConsumerID: 0x7eb8b255e1d4bf64333186dcde7d703e5d34da3a Data: 40.3KiB/15.7KiB Tokens: 0.000000
[ID: f53491bb-f0d9-4d8e-a5b3-a052c9090811] ConsumerID: 0xdfdb9b72a1a26539902f4ab9177d1752accbf3fd Data: 449.5KiB/406.0KiB Tokens: 0.000000
[ID: 1ed455e7-8a1f-4ba5-b550-c0848bda0d01] ConsumerID: 0x81538d388bf57aa38490c30b07040c9e948f6cbf Data: 245.5MiB/10.3MiB Tokens: 0.000000
[ID: d74882b4-0b59-4c93-bed7-437af162c0d9] ConsumerID: 0x690c4f186d2ad76e6faa0b6f5bf691abf000ed05 Data: 454.2KiB/449.5KiB Tokens: 0.000000
[ID: 865a9501-c7f6-458f-bae2-1085755ca4e8] ConsumerID: 0xdfdb9b72a1a26539902f4ab9177d1752accbf3fd Data: 243.1KiB/146.5KiB Tokens: 0.000000
[ID: 46f8bcee-34d5-48a0-b985-3250b2b9d093] ConsumerID: 0xe88df80ba994a662fec73a104777f018a09d4491 Data: 440.5KiB/1.3MiB Tokens: 0.000000
[ID: db8992a3-c420-4ef8-887d-a79e52f211a1] ConsumerID: 0x2ecefb27ee3693ce7cd47a94ec0325bd654df6ad Data: 448.7KiB/403.0KiB Tokens: 0.000000
[ID: a479d181-850f-4055-862c-9a507cf5336b] ConsumerID: 0x75eb802ac33fd2c5c3405aa338551bd78c7e3c4c Data: 411.0MiB/14.7MiB Tokens: 0.000000
[ID: 4164ba22-7470-420c-ab6a-870e04afa4a5] ConsumerID: 0xb5a2f38e62c5b3e73717ab0b38379d521f6f5aca Data: 454.8KiB/431.7KiB Tokens: 0.000000
[ID: 8e41b5f7-6665-44cf-b6e9-d7c246cffc12] ConsumerID: 0xebd240db9bcc4d79d2e3113ad5303e622897f1bf Data: 40.0KiB/15.7KiB Tokens: 0.000000
[ID: a52aa196-713c-460f-8925-02b8a5c1ca14] ConsumerID: 0xac96e76e3c9b66e02ba3a87799bde2f6012edff3 Data: 429.0KiB/402.9KiB Tokens: 0.000000
[ID: d2a62669-09b5-4bfa-9056-f7cf22aef21b] ConsumerID: 0x652b7ed8143adf41b7be437d490589ccad02bf31 Data: 429.1MiB/15.7MiB Tokens: 0.000000
[ID: e8354573-f092-4585-86d2-f0ab9d73e663] ConsumerID: 0xa2a29af26f6b0e04d901d1bee0ffaefdcaded971 Data: 447.3KiB/422.9KiB Tokens: 0.000000
[ID: 8363c426-fa7f-45ec-985c-68fece902f8c] ConsumerID: 0xc72a9750531f70e0a4ae627073616248b456361d Data: 42.2KiB/16.9KiB Tokens: 0.000000
[ID: 1801ebef-886e-4888-bca1-15a7c72eb0b4] ConsumerID: 0xac83c06eff343ebd3aa137cefca957aa2cf23237 Data: 425.4MiB/14.0MiB Tokens: 0.000000
[ID: 9439e226-e3d3-42cf-ba67-c3448442595a] ConsumerID: 0xdfdb9b72a1a26539902f4ab9177d1752accbf3fd Data: 423.0KiB/423.1KiB Tokens: 0.000000
[ID: a1b62bbc-09b2-4cd6-9d02-0deb5e637757] ConsumerID: 0x318f500d5a9e78459684055b911f07e76aa70669 Data: 453.2KiB/439.3KiB Tokens: 0.000000
[ID: 91bb414d-8918-4dd7-a1b5-5aaab159587e] ConsumerID: 0x56da8fb62b3759bfa9d89a09a7aa9846e1922a19 Data: 39.8KiB/15.5KiB Tokens: 0.000000
[ID: e24a26cd-ccf1-405a-9f75-824d8438d581] ConsumerID: 0xaee847c79c8721f58b0e2b677f33c8018dd2032f Data: 452.5KiB/439.1KiB Tokens: 0.000000
[ID: 68f09390-9be0-44c8-8b6d-ba37198207b2] ConsumerID: 0xa050be13ea2ad911a162de98b246d3657cea094d Data: 453.9KiB/440.3KiB Tokens: 0.000000

**Comment by mitjats** [comment]
Hope that someone can help.
Actually I have 191 sessions, 80 unique clients, 24.78GB transfered data but still no rewards.

**Comment by Donatas-MN** [comment]
@mitjats Do you see earnings in "Total earnings" in NodeUI and MystNodes.com?

**Comment by mitjats** [comment]
From what I remember I got (and have) 0.067842 MYST earning in my mystnodes.com from start. If not from day one, there was no change in earning since long time.

**Comment by Donatas-MN** [comment]
Can you send a screenshot of your current NodeUI dashboard?

**Comment by mitjats** [comment]
https://www.icloud.com/iclouddrive/0f16d5y8JwCcCgzPEd8Vo5BeQ#Schermata_2021-12-01_alle_08.41
this is the situation dec 1th

https://www.icloud.com/iclouddrive/0f16d5y8JwCcCgzPEd8Vo5BeQ#Schermata_2021-12-01_alle_08.41
and this is right now

**Comment by Donatas-MN** [comment]
Looks like you're using older Node version, can you try to update it to newest version?

After that, please send a screenshot of Settings page in NodeUI.

Which browser are you using? Can you try different device/browser?

**Comment by mitjats** [comment]
I tried to made the upgrade at the beginning, but the curl command hasn't worked on my raspberry pi, I don't know why, cause I haven't investigated, but now it seems to work and I upgraded the node to the last stable version.
I'm browsing with Firefox DevEd 95 MacOs.
This is the cache output:

- - - 

sudo apt-cache policy myst
myst:
  Installed: 1.0.14+build428056802+focal
  Candidate: 1.0.14+build428056802+focal
  Version table:
 *** 1.0.14+build428056802+focal 500
        500 http://ppa.launchpad.net/mysteriumnetwork/node/ubuntu focal/main armhf Packages
        100 /var/lib/dpkg/status
- - - 
And this is the last screenshot after the upgrade.
https://www.icloud.com/iclouddrive/079nRojBCVunIqgR6vxuxsQXw#Schermata_2021-12-20_alle_10.06

**Comment by Donatas-MN** [comment]
And if you try with a different browser?

**Comment by mitjats** [comment]
I don't think it's a browser related problem.
This is the info output made after 2 new sessions.
- - -
myst account info
[SECTION] General account information:
[INFO] Using identity: 0x4...
[INFO] Registration Status: Registered
[INFO] Channel address: 0x7...
[INFO] Balance: 0.000000
- - -

**Comment by mitjats** [comment]
no advise?

**Comment by Donatas-MN** [comment]
Does it show the same earnings (0.067842 MYST) on MystNodes also? And on a different device/browser? 

Can you submit logs from NodeUI? Just click "bug" button and send a message from there, logs will be attached.

**Comment by mitjats** [comment]
I just checked from my iPad Safari and got the same results.
I sent the report from "bug".

**Comment by Donatas-MN** [comment]
Looks like earnings are fine, but they are not synced with NodeUI for some reason. We will investigate and get back.

Logs:
[logs.zip](https://github.com/mysteriumnetwork/node/files/7761735/logs.zip)

**Comment by mitjats** [comment]
I don't understand so much but there is some incongruences in this file log.
The time stamp is wrong, and when I started the node a month ago, I checked that the correct (Italy) time/zone, date and time were selected. But I noted that in all my logs the date seems always the same like in this log file. This erratic problem was not always present and makes me believe that it was just an initial setup glitch, but seems that's not the case.

**Comment by Donatas-MN** [comment]
@mitjats It might be that something got corrupted when you started your node thus earnings are not transferred to your payment channel. Can you try to start a completely new node to see if it works fine?

We will compensate those lost MYST to the new node (looks like it's ~3.2MYST). Just let us know your new node's payment channel (not the identity) and we will top-up it.
![Pasted_Image_2021-12-29_16_27](https://user-images.githubusercontent.com/91055712/147672357-e66b6eab-dfc9-4973-9de7-125eaa946671.jpg)

**Comment by mitjats** [comment]
I will try and tell you the result.

**Comment by Donatas-MN** [comment]
@mitjats any luck?

**Comment by mitjats** [comment]
How can I stat a fresh new node?
I searched the faqs and documentation, but find no clue on it.

**Comment by Donatas-MN** [comment]
@mitjats what device are you running your node on?

**Comment by mitjats** [comment]
on a raspberryPi 3

**Comment by Donatas-MN** [comment]
@mitjats just take out SD card, reflash it with the newest Node version and put it back to your RPi: https://mystnodes.com/onboarding/rpi/install/0 

This will start completely new node.

**Comment by Donatas-MN** [comment]
Yes, so you get new identity.

**Comment by mitjats** [comment]
ok, I flashed again the card and started a fresh new node.
I made all the upgrades and now seems that all is ok.
Hope that this time the earnings will work.

**Comment by Donatas-MN** [comment]
Great, thanks! Let us know if everything works fine after a few days.

**Comment by mitjats** [comment]
ok, I'll be back in few days.

**Comment by mitjats** [comment]
Now seems ok. The earnings start to be registered.

**Comment by Donatas-MN** [comment]
Great, thanks for confirming!

**Comment by mitjats** [comment]
this is my new payment channel for the previous earnings:
0x5f220B5a2b36C577AB991cf3082E8A3e172aCd93

**Comment by Donatas-MN** [comment]
Sure, should reach you in a couple of days.

**Comment by mitjats** [comment]
thank you

**Comment by mitjats** [comment]
Now seems that my web ui does not work properly. After the first few days, the recorded data of the various sessions, are no longer updated in the upper cards section. They are at zero for all except for the earnings that is (rounded) correct. The graph at the bottom is updated regularly.
Im on macOS and track my node with Firefox, but tried with Safari and Chrome and Safari iOS with identical results.
It happened in the past few times, but after a minute or two, the data in the upper cards revert back to normal, but this time the bug seems permanent.

**Comment by Donatas-MN** [comment]
That's a known bug in a new Node version. Will be fixed next week.

**Comment by mitjats** [comment]
ok.

**Comment by mitjats** [comment]
Thank you for the previous earnings, but ... what a fee commission if in percentage.

**Comment by Donatas-MN** [comment]
> Thank you for the previous earnings, but ... what a fee commission if in percentage.

What do you mean?

**Comment by mitjats** [comment]
in percentage is 24,83%

**Comment by Donatas-MN** [comment]
The network fee is fixed at 20%, the rest is Polygon blockchain transaction fees. With current auto-settlement of 5MYST it should be much less than 4,83%.

**Comment by mitjats** [comment]
Ok, then I’ll be wait until I reach at least 1000 MYST :-)

**Comment by Donatas-MN** [comment]
Just to clarify :) Auto-settlement kicks-in at 5MYST, so from that 20% fee will be deducted as network settlement fee (1 MYST in this case) plus blockchain fees (currently up to 0.1 MYST). So all in all, you would receive ~3.9 MYST which you can withdraw to your wallet.

**Comment by mitjats** [comment]
Yes, I understand. No problem.

**Comment by mitjats** [comment]
It is possible to change the auto settlement limit?

**Comment by Donatas-MN** [comment]
> It is possible to change the auto settlement limit? I see the issue #4577 but haven't been able to change anything on the web interface.

Do you want to reduce or increase it?

**Comment by mitjats** [comment]
Increase it. But I think of a command in CLI or a sort of slider in web interface to trim up or down

**Comment by mitjats** [comment]
What's going on? Is it normal?
<img width="367" alt="Schermata 2022-02-17 alle 08 20 36" src="https://user-images.githubusercontent.com/8850813/154425621-73c43b4e-40ac-49c7-9205-604a717066f6.png">

**Comment by Donatas-MN** [comment]
Yes, it's most probably a video streaming session.

**Comment by mitjats** [comment]
Now is 22,5 hrs and still going.

**Comment by Donatas-MN** [comment]
That's totally normal, it's most probably whitelisted streaming traffic from Mysterium partners. If you keep your node online 24/7, you can expect even more coming in.

**Comment by mitjats** [comment]
Yes, it's always on. I never shut down it.

**Comment by Donatas-MN** [comment]
That's why you started to get such connections :)

**Comment by mitjats** [comment]
Is there a problem with the network?
I can't reach my node on the lan from 2 days and the same if I check on mystnodes.com.

**Comment by Donatas-MN** [comment]
Strange, we didn't have any major issues recently. Can you access NodeUI? Try to restart the node.

**Comment by mitjats** [comment]
You where right. Restarting the node comes back to normal.

**Comment by mitjats** [comment]
I tried to upgrade the node version as per email I received, but couldn't upgrade to 1.6.3. The last version I see is 1.6.11, which I'm currently on. I think that 1.6.11 is higher than 1.6.3 or am I wrong?
I ssh to my myst node to do this, and made other updates. Then I would to reboot the node from ssh, but the answer I got is "Failed to set wall message, ignoring: Interactive authentication required.
Failed to reboot system via logind: Interactive authentication required.
Failed to open initctl fifo: Permission denied
Failed to talk to init daemon."
I searched the faq's but haven't find any instruction on how to do this.
How can I reboot from sw and not pulling out the dc plug?

Thanks for your help.
